Podcast/FollowerFollowingViewController.swift

```diff
@@ -0,0 +1,110 @@
+//
+//  FollowerFollowingViewController.swift
+//  Podcast
+//
+//  Created by Drew Dunne on 9/10/17.
+//  Copyright © 2017 Cornell App Development. All rights reserved.
+//
+
+/**
+ * This view controller is for both displaying your followers and whom you follow.
+ */
+
+import UIKit
+import NVActivityIndicatorView
+
+class FollowerFollowingViewController: UIViewController, UITableViewDataSource, UITableViewDelegate {
+    
+    let cellIdentifier = "searchUsersCell"
+    
+    var usersTableView: UITableView!
+    var loadingActivityIndicator: NVActivityIndicatorView!
+    var refreshControl: UIRefreshControl!
+    
+    var users: [User] = []
+    var followersOrFollowings: UserFollowsType!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = .podcastWhiteDark
+        title = followersOrFollowings == .Followers ? "Followers" : "Following"
```

Comment:
this is actually probably the default toString .. if not let's make this a variable on the enum

Reply:
There is no toString function....